src/core/markovProcessGraph.js

```diff
@@ -172,7 +207,14 @@ const FIBRATION_EDGE = EdgeAddress.fromParts([
 ]);
 const EPOCH_PAYOUT = EdgeAddress.append(FIBRATION_EDGE, "EPOCH_PAYOUT");
 const EPOCH_WEBBING = EdgeAddress.append(FIBRATION_EDGE, "EPOCH_WEBBING");
-const EPOCH_RADIATION = EdgeAddress.append(FIBRATION_EDGE, "EPOCH_RADIATION");
+const USER_EPOCH_RADIATION = EdgeAddress.append(
+  FIBRATION_EDGE,
+  "USER_EPOCH_RADIATION"
+);
```

Comment:
Let's remove?

Reply:
If we remove this, then the out-edges from a user epoch node are:

  - a payout edge with mass β
  - webbing edges with mass γ₁ + γ₂
  - base edges (to/from other contributions) with remaining mass

But in the case where there are no base edges for a user-epoch (which
will be quite common), this means that 1 − β − (γ₁ + γ₂) of the mass is
unaccounted for. Where should it go—to the seed? Anywhere that you send
it seems to create strange incentives. So we’d like to hold off on this.